sys/src/9/386/pci.c

```diff
@@ -193,7 +193,6 @@ pcilscan(int bno, char *path, Pcidev** list)
 				}
 				break;
 
-			case 0x00:
```

Comment:
This was part of PR #180 (Some virtio-pci devices (e. g. 9p)  have ccrb = 0x00, their BARs and sizes also shouldbe  picked up here) which was reverted (probably by omission). It has to do with virtio of course.